app/controllers/concerns/remember_device_concern.rb

```diff
@@ -6,7 +6,7 @@ def save_user_opted_remember_device_pref
   end
 
   def save_remember_device_preference
-    return unless params[:remember_device] == 'true'
+    return unless params[:remember_device] == '1'
```

Comment:
> I don't remember if we made this "deploy friendly" but we could check for both `'1'` and `'true'` if we were worried about a smooth transition during deploy

Yeah, seems like a good idea, since these MFA screens would be a pretty hot path.

Reply:
Added in d5c82ae.